.storybook/stories/modal/sidebar.stories.ts

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { ClrModalModule, ClrSidebar, commonStringsDefault } from '@clr/angular';
+import { action } from '@storybook/addon-actions';
+import { moduleMetadata, StoryFn, StoryObj } from '@storybook/angular';
+
+import { CommonModules, removeFocusOutline } from '../../helpers/common';
+
+export default {
+  title: 'Modal/Sidebar',
+  decorators: [
+    moduleMetadata({
+      imports: [...CommonModules, ClrModalModule],
+    }),
+  ],
+  component: ClrSidebar,
+  argTypes: {
+    // inputs
+    clrSidebarCloseButtonAriaLabel: { type: 'string', defaultValue: commonStringsDefault.close },
+    clrSidebarLabelledById: { defaultValue: '' },
+    clrSidebarSize: {
+      defaultValue: 'md',
+      control: { type: 'radio', options: ['sm', 'md', 'lg', 'xl', 'full-screen'] },
+    },
```

Comment:
Otherwise it's empty and not working.
```suggestion
    clrSidebarSize: {
      defaultValue: 'md',
      options: ['sm', 'md', 'lg', 'xl', 'full-screen'],
      control: { type: 'radio' },
    },
```

Reply:
Good catch, thanks!

Reply:
Done